internal/test/testing.go

```diff
@@ -108,6 +108,7 @@ type Test struct {
 	AppRoleAssignedToClient                   *msgraph.AppRoleAssignedToClient
 	AuthenticationMethodsClient               *msgraph.AuthenticationMethodsClient
 	B2CUserFlowClient                         *msgraph.B2CUserFlowClient
+	ChatClient                                *msgraph.ChatClient
```

Comment:
You will also need to instantiate & configure this client in the NewTest method

Reply:
I can't believe I missed that.
Implemented here: [Commit: 649bd16765c3c72b91d8193841c3a0dc5c0a9ab0](https://github.com/manicminer/hamilton/pull/219/commits/649bd16765c3c72b91d8193841c3a0dc5c0a9ab0)